shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -24,7 +25,7 @@
  */
 public class TextInputPlugin {
     @NonNull
-    private final FlutterView mView;
+    private final View mView;
```

Comment:
Why are we reducing the specificity here? It doesn't look like we currently use anythingspecific to FlutterView, but it also doesn't seem like this class would be all that useful for a non-FlutterView derived class (and we probably shouldn'tencourage people to use it that way).

Reply:
We now have 2 different definitions for FlutterView, so the existing type is no longer enough.

Reply:
Will this ever possibly need to work with a FlutterView (new or old) directly?  I suppose in this refactoring if we needed to send a message we could have a new constructor that takes a messenger as a parameter?

Reply:
I don't think any of us know what the future holds, but I have no reason to think we'll need a more specific type.  In the worst case, we can apply an Interface to both versions of FlutterView, but that is a last resort.

Reply:
I'm ok with the idea that we could always add another constructor to this class that would take in the more specific things it needs.  Looking at FlutterView in both cases, it's not as if there's some property this needs.